Adds milliseconds to interval options

ycombinator opened the pull request:
Closes #6559

epixa commented on src/ui/public/utils/__tests__/parse_interval.js:
```diff
@@ -64,8 +67,5 @@ describe('parseInterval', function () {
     expect(duration).to.not.be.ok();
-
-    duration = parseInterval('ms');
-    expect(duration).to.not.be.ok();
```
Removing this test makes sense, but the `parseInterval('ms')` case should probably be added to the test block above (lines 47-59).  I assume it behaves the same as d, m, y, M.

spalger commented:
LGTM

epixa commented:
I added one note about a missing test, otherwise this is looking good.

ycombinator commented:
> I added one note about a missing test, otherwise this is looking good.

@epixa I thought I added that already as part of https://github.com/elastic/kibana/pull/6796/commits/261822063a9bf9f3494557938ba16bcc20c6f786. Or are you referring to something else?

ycombinator commented:
@epixa Never mind, I see what you are saying now. Will add the test.

epixa commented:
I'm seeing a weird behavior with milliseconds on a larger time range.  If I do a date histogram on Count with a quick time range of `Last 1 Hour`, the bucket interval is automatically adjusted to 30 seconds regardless of whether I select millisecond or second (which I'd expect).  However, I'd expect the details of the resulting graph to be essentially identical since the bucket range was adjusted to the same amount, but they're not.  The milliseconds graph is showing 0 for every rung on the y-axis (and on the tooltip mouseover), while the seconds graph is at least showing decimals:

Seconds:
![screen shot 2016-04-06 at 4 21 26 pm](https://cloud.githubusercontent.com/assets/53814/14331535/bcc525de-fc13-11e5-994b-d7cd5347a53c.png)

Milliseconds:
![screen shot 2016-04-06 at 4 21 37 pm](https://cloud.githubusercontent.com/assets/53814/14331537/c084637e-fc13-11e5-9361-968a65880f51.png)

I'm honestly not sure if that makes sense or not, but it seems like a bug.

ycombinator commented:
@epixa I checked and the behavior you reported exists in `master` too. I tend to agree that it looks like a bug so I've created this issue for it: https://github.com/elastic/kibana/issues/6812

rashidkpc commented:
Decimal precision is set to 3 places by default. So your count at 1s was 0.233, divide that by 1000 to get to milliseconds and you're at 0.000233, which rounds down to 0. Increase your decimal precision in advanced settings and that should take care of this. @epixa @ycombinator

epixa commented:
Makes sense. Thanks @rashidkpc.

epixa commented:
LGTM
